neo/io/nixio.py

```diff
@@ -397,9 +397,28 @@ def _nix_to_neo_analogsignal(self, nix_da_group):
         else:
             t_start = create_quantity(timedim.offset, timedim.unit)
 
+        # find annotations with length equal to signal length and turn them into array-annotations
```

Comment:
This block of code (until `del neo_attrs[key]`) is repeated many times. For maintainability, it would make sense to pull it out as a function.

Reply:
I went with Julia's suggestion (https://github.com/NeuralEnsemble/python-neo/pull/662#issuecomment-481376527) which makes this entire block unnecessary.